src/shared/components/Table/Table.module.css

```
@@ -0,0 +1,220 @@
+/* ==================== Container & Table Base ====================*/
```

Stylelint (17.11.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)